src/components/calcite-accordion-item/calcite-accordion-item.scss

```diff
@@ -1,8 +1,4 @@
-//ie11 fixes
-:host-context([theme="dark"]) {
-  @include calcite-theme-dark();
-}
-
+// host-content() is being used to support IE11 styling in this component.
```

Comment:
It would be nice to get rid of host context here too since it is the last place using it.